src/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "ux-aspects",
+  "version": "1.1.0-beta.1",
+  "description": "Open source user interface framework for building modern, responsive, mobile big data applications",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/UXAspects/UXAspects.git"
+  },
+  "keywords": [
+    "ux",
+    "aspects",
+    "ui",
+    "framework",
+    "angular",
+    "angularjs",
+    "bootstrap"
+  ],
+  "author": "Ashley Hunter",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/UXAspects/UXAspects/issues"
+  },
+  "homepage": "https://uxaspects.github.io/UXAspects",
+  "dependencies": {
+    "chart.js": "^2.5.0",
+    "ng2-charts": "^1.5.0",
+    "ngx-bootstrap": "^1.6.6"
+  },
+  "devDependencies": {
+    "@types/chart.js": "^2.4.2",
```

Comment:
Didn't add typescript as technically they can use plain JS or Dart either. Other libs like ngx-bootstrap don't appear to include it as a dependency. I added the type definitions anyway because you can now use the Typescript language service to give hints in regular JS files.